src/org/usfirst/frc/team1518/robot/Robot.java

```diff
@@ -38,40 +33,36 @@
     Solenoid sol1, sol2;
 
     int session;
-    Image frame;
-    AxisCamera camera;
     
     Command grabBall;
     Command shootBall;
     Command reverseDrive;
     
+    private int mode = 1; // initialize default mode
+    private SendableChooser autoChoice;
+
     
     public void robotInit() {
 
-        frame = NIVision.imaqCreateImage(NIVision.ImageType.IMAGE_RGB, 0);
+        autoChoice = new SendableChooser();
+        autoChoice.addDefault("Auto OFF", 0);
+        autoChoice.addObject("Auto Mode 1", 1);
+        autoChoice.addObject("Auto Mode 2", 2);
+        autoChoice.addObject("Auto Mode 3", 3);
+        SmartDashboard.putData("Autonomous Selector", autoChoice);
```

Comment:
Because the Java SmartDashboard lib doesn't support arrays or objects, I think this will need to be a series of calls that put a boolean used in a set of buttons. Something like:

```
SmartDashboard.putBoolean("Auto OFF", false);
SmartDashboard.putBoolean("Auto Mode 1", false);
SmartDashboard.putBoolean("Auto Mode 2", false);
...
```

Then, we can read that button value later with `boolean buttonValue = SmartDashboard.getBoolean("Auto OFF", false);` probably in an init we'd loop through the buttons to find out which one is on. They're not mutually exclusive, though we could code for that so that turning one on turns all the others off.

See https://wpilib.screenstepslive.com/s/4485/m/24192/l/272692-using-the-labview-dashboard-with-c-java-code